### PR TITLE
[COE] Update confirmation page

### PR DESCRIPTION
**Pull request by jerekshoe**
## Description
Updates the confirmation page to closer match the [mockup](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/137667003/specification/sitemap).

## Original issue(s)
department-of-veterans-affairs/va.gov-team#33092

## Acceptance criteria
- [x] Confirmation page closely matches mockup

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/lgy/coe/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -12,41 +15,76 @@ export class ConfirmationPage extends React.Component {
...
 
   render() {
-    const { submission, data } = this.props.form;
-    const { response } = submission;
-    const name = data.veteranFullName;
+    // const { submission, data } = this.props.form;
+    // const { response } = submission;
+    // const name = data.veteranFullName;
+
+    const name = {
```
Comment (cohnjesse) [inline src/applications/lgy/coe/containers/ConfirmationPage.jsx:22]: Are we meaning to not use the form data and instead make this static?

File: src/applications/lgy/coe/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -12,41 +15,76 @@ export class ConfirmationPage extends React.Component {
...
+      first: 'Robert',
+      middle: 'John',
+      last: 'Jones',
+      suffix: 'Jr.',
+    };
+
+    const response = {
```
Comment (cohnjesse) [inline src/applications/lgy/coe/containers/ConfirmationPage.jsx:29]: Same question as above

File: src/applications/lgy/coe/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/lgy/coe/containers/ConfirmationPage.jsx]: Fixed

File: src/applications/lgy/coe/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (jerekshoe) [reply src/applications/lgy/coe/containers/ConfirmationPage.jsx]: fixed

**Review by cohnjesse — APPROVED** [review]
LGTM

**Pull request merged**